apps/insights/src/components/PriceFeed/Chart/use-chart-toolbar.tsx

```diff
@@ -0,0 +1,58 @@
+import { parseAsStringLiteral, useQueryState } from "nuqs";
+
+export const RESOLUTIONS = ["1s", "1m", "5m", "1H", "1D"] as const;
+export type Resolution = (typeof RESOLUTIONS)[number];
+
+export const LOOKBACKS = ["1m", "1H", "1D", "1W", "1M"] as const;
+export type Lookback = (typeof LOOKBACKS)[number];
```

Comment:
I really don't like the name "lookbacks" for this -- if we do the auto-resolution adjust, this is more accurately a "snap to" than a lookback.  Even now I think this is effectively behaving as a "snap to"

Reply:
yep yep, i don't have a strong opinion here. another candidate was `INTERVAL`, but i found that it could be easily confused with `RESOLUTION`. what do you think?

Reply:
the best I have in mind is `SNAP_TO_PERIODS`.  I'm sure there's a better name, but `INTERVAL` doesn't seem clear to me either.  Maybe @alexcambose has some ideas?

Reply:
`QUICK_SELECT_WINDOWS` maybe?